### ISSUE TITLE
Ravencoin mining T-rex 0.21.5/4/3 error

### ISSUE THREAD

**Issue by hsof168** [trexminer/T-Rex#555]
I tried to mine ETH and FIRO it works without issue, but when I try to mine RVN the program gave me an error.
Tried 3 versions still the same. Tried Driver v471.11 and v471.41 same result.
Deleted trex-1.dll and trex-2.dll in the folder and restart Trex Miner program also gave me same error. 

Computer Spec :
Gigabyte RTX 2080 Gaming OC
Driver Nvidia v471.41
T-rex v0.21.3, also tried 0.21.4 and 0.21.5 same issue
Windows 10 v20H2 OS Build 19042.1110

![Capture01](https://user-images.githubusercontent.com/88285286/127779488-ac1dfc79-754a-48b8-b7a2-0686dcf280c0.JPG)

Anybody can help?

**Comment by trexminer** [comment]
Error code 5 is `ERROR_ACCESS_DENIED`, so my guess is it's either an antivirus or file permission issue.

**Comment by hsof168** [comment]
Is it possible antivirus or file permission issue just for the ravencoin (kawpow) mining? because ethash and mtp works fine.

**Comment by trexminer** [comment]
Yes, it is possible. `trex-1.dll` and `trex-2.dll` are only loaded for progpow algorithms.

**Comment by aloyco** [comment]
Did you try turning off your firewall? I remember some from taiwan had problems connecting to ethermine/flypool servers so it might be ISP too.

**Comment by hsof168** [comment]
@aloyco I did turn off the firewall, the miner is working for eth and firo. If I turn on my firewall eth and firo doesn't work.

I am using Norton for this Gaming PC. If I am browsing or gaming I will turn on the Norton, if mining then I will turn it off. Already apply settings to not scan both local/temp folder and the t-rex miner folder. Will try something else. Cheers
